lighthouse-core/lib/traces/tracing-processor.js

```diff
@@ -177,57 +121,60 @@ class TraceProcessor {
   /**
    * Calculates the maximum queueing time (in ms) of high priority tasks for
    * selected percentiles within a window of the main thread.
-   * @see https://docs.google.com/document/d/1b9slyaB9yho91YTOkAQfpCdULFkZM9LqsipcX3t7He8/preview
-   * @param {!traceviewer.Model} model
-   * @param {{traceEvents: !Array<!Object>}} trace
+   * @see https://docs.google.com/document/d/18gvP-CBA2BiBpi3Rz1I1ISciKGhniTSZ9TY0XCnXS7E/preview
+   * @param {!TraceOfTabArtifact} tabTrace
    * @param {number=} startTime Optional start time (in ms) of range of interest. Defaults to trace start.
    * @param {number=} endTime Optional end time (in ms) of range of interest. Defaults to trace end.
    * @param {!Array<number>=} percentiles Optional array of percentiles to compute. Defaults to [0.5, 0.75, 0.9, 0.99, 1].
    * @return {!Array<{percentile: number, time: number}>}
    */
-  static getRiskToResponsiveness(model, trace, startTime, endTime, percentiles) {
+  static getRiskToResponsiveness(tabTrace, startTime, endTime, percentiles) {
+    const navStart = tabTrace.navigationStartEvt.ts;
+    const traceStart = tabTrace.earliestTraceEvt.ts;
+    const traceEnd = tabTrace.latestTraceEvt.ts;
+
     // Range of responsiveness we care about. Default to bounds of model.
-    startTime = startTime === undefined ? model.bounds.min : startTime;
-    endTime = endTime === undefined ? model.bounds.max : endTime;
+    startTime = startTime === undefined ? (traceStart - navStart) / 1000 : startTime;
```

Comment:
> getting everything after navStart is our typical use case right now

though that's not completely true. It looks like all of our calls have to deal with events earlier than they care about anyways, so it doesn't seem harmful to default to all events, even before navStart